### PR TITLE
[URLIZATION] don't delete a letter from url when it has an accent

### PR DESCRIPTION
**Pull request by cmarianhodis**
Related issue: https://github.com/backbee/backbee-standard/issues/748

### CODE REVIEW

File: StringUtils.php [hunk 1]
```diff
@@ -147,13 +148,43 @@ public static function urlize($str, $options = null, $charset = 'UTF-8')
...
     /**
+     * Replace special characters in a given string
+     *
+     * @access public
+     * @param string $str The string in which to replace special characters
+     * @return string The string with the special characters replaced
+     */
+    public static function replaceSpecialChars($str)
+    {
+        $specialCharsArray = [
```
Comment (eric-chau) [inline StringUtils.php]: Can you declare this array as a static attribute of `StringUtils` please?

**Comment by eric-chau** [comment]
Good job @cmarianhodis. :+1: thank you!

**Comment by eric-chau** [comment]
Also if you can add some Tests it would be nice ;) just 2 or 3 cases will be enough.

**Comment by cmarianhodis** [comment]
@eric-chau added testcases and made the modification per suggested

**Comment by eric-chau** [comment]
Good job thank you @cmarianhodis!

**Pull request merged**